routing/routing.xml

```diff
@@ -23,29 +23,234 @@
+        
+        <select value="-1" t="oneway" v="-1"/>
+            
+        
+    ## 4) Priority (specified on ways, between 0 and 1)
+    
+        Priority is a preference multiplicator (between 0 and 1), which can only be used to _de_prioritize routes.
+        The formula used is 
+        `minimum(speed * multiplier, speed)`
+
+
+    ## 5) Obstacle (specified on points, penalty time in seconds or -1 if blocked)
+    
+        This block evaluates points encountered on the road.
+        The block specifies if this obstacle can be passed and eventually a penalty for passing it.
+        
+        A value of "-1" is given if the obstacle can not be passed;
+        A value of "0" is given if the obstacle can be passed without problem.
+        A penalty in seconds is given if the obstacle slows you down. (E.g. a kerb might slow dow 
+        
+        This value is only used by the routing engine
+        
+
+        E.g. a kerb is a showstopper for a wheelchair
+        <routingProfile name="wheelchair">
+            <point attribute="obstacle>
+                <select value="-1" t="barrier" v="kerb" />
+            </point>
+        </routingProfile>    
+        
+        
+        E.g. a kerb causes a slight delay for cyclists
+        <routingProfile name="bicycle">
+            <point attribute="obstacle>
+                <select value="5" t="barrier" v="kerb" />
+            </point>
+        </routingProfile>  
+        
+
+    ## 6) Obstacle_time (specified on points, penalty time in seconds)
+
+        Gives a slowdown time for an obstacle in seconds.
+        In contract to `obstacle`, this value is only used to calculate an ETA and show it on the UI once the route is calculated.
+        In other words, the values here do not influence routing.
+        
+        
+
+    ## 7) Penalty_transition (specified on ways, penalty time in seconds)
+
+        When driving a car, changing from a motorway to a trunk slows you down: you should already slow down 500 meters before the trunk.
+        This has a penalty which is taken into account by (only) the navigation algorithm.
+        This is done by taking the difference between two penalties.
+        The formula used is `minimum(0, penalty_new_way - penalty_current_way)`.
+        
+        E.g. 
+        <way attribute="penalty_transition">
+            <select value="0" t="highway" v="motorway"/>
+            <select value="5" t="highway" v="motorway_link"/>
+            <select value="20" t="highway" v="tertiary"/>
+        </way>
+        
+        
+        This means that, going from a `motorway_link` to a `tertiary`, the penalty given will be
+        `minimum(0, 20 - 5)` or `15 seconds`.
+        
+        On the other hand, going from motorway to motorway incurs no penalty, neither does going from a motorway_link to a motorway.
+
+    ## 8) obstacle_srtm_alt_speed (specified on ways, in km/h)
```

Comment:
These values are not alternative speeds measured in km/h, higher values mean higher penalty, not higher speed.